### DISCUSSION TITLE
Is it possible to merge two trilium notes databases?

### DISCUSSION THREAD

**Discussion by quakeboy** [TriliumNext/Trilium#2997] [Q&A]
I was using one on my windows machine, and then started one more on my mac. Can I merge them?

In the future I'll try to get a server installed somewhere on the cloud on my private server. But for now is there a way to do this? Thank you in advance.

**Comment by zadam** [comment]
You can export the whole Trilium database as a ZIP file which can be imported into another database: 

![image](https://user-images.githubusercontent.com/617641/179855648-fbc26db8-b408-4b52-8ca8-8e26cc2c0ccc.png)

The only thing which you'll lose is revision history, which isn't part of the ZIP export.

**Comment by EdouardAP** [comment]
Exporting a Trilium database and importing into another database will not delete duplicate entries. The information in the imported database will be added to the original.
